packages/remark-mdx/readme.md

```diff
@@ -22,6 +22,16 @@ It’s used in [MDXjs][].
 npm install remark-mdx
 ```
 
+### Installing Other Plugins
+
+You'll probably need other plugins such as [**remark-parse**][remark-parse].
```

Comment:
I'm not sure what you mean by this.
MDX v1 is remark parse 8 with an additional JSX parser, MDX v2 is remark parse 9 with additional parsers.
remark parse doesn't need to be installed again.

Reply:
I'm talking about **remark-mdx** plugin specifically. [It seems](https://github.com/mdx-js/mdx/blob/e36caef9eee3a6f472b268c1160e2deabe53187e/packages/remark-mdx/package.json#L40-L44) that it should be used with **remark-parse** (or some other markdown parser).

Reply:
/cc @wooorm thoughts on how this should be noted?